### PR TITLE
workaround for message count

### PR DESCRIPTION
**Pull request by guykisel**

### CODE REVIEW

File: inlineplz/interfaces/github.py [hunk 1]
```diff
@@ -181,6 +181,9 @@ def post_messages(self, messages, max_comments):
                     msg_position
                 )
             except github3.GitHubError as err:
+                # workaround for our diff not entirely matching up with github's diff
+                # we can end up with a mismatched diff if the branch is old
+                messages_to_post -= 1
```
Comment (riot-mraviv) [inline inlineplz/interfaces/github.py]: Not sure why it is important to decrement messages_to_post here but not in for example line 169.

Comment (guykisel) [reply inlineplz/interfaces/github.py]: so the idea is that i want to have a nonzero `messages_to_post` count if there are still known errors in the diff. if a message is detected as a duplicate that means that the error there is still valid. however in the case of a githuberror, most likely the error is not actually part of the diff and is not valid.

Comment (riot-mraviv) [reply inlineplz/interfaces/github.py]: Oh, got it

Comment (guykisel) [reply inlineplz/interfaces/github.py]: good call

**Pull request merged**